clap_man/src/man.rs

```diff
@@ -0,0 +1,62 @@
+pub(crate) fn bold(input: &str) -> String {
+    format!(r"\fB{}\fR", input)
+}
+
+pub(crate) fn italic(input: &str) -> String {
+    format!(r"\fI{}\fR", input)
+}
+
+pub(crate) fn escape(str: &str) -> String {
+    str.replace("-", r"\-")
+}
+
+pub(crate) fn indent(level: i8, input: &str) -> String {
+    format!(".RS {}\n{}\n.RE", level, input)
+}
+
+pub(crate) fn section_title(title: &str) -> String {
+    format!(".SH \"{}\"\n", title.to_uppercase())
```

Comment:
I'm torn.  I'm assuming its pretty much required that a `section_title` have a newline but its throwing me off when reading the code to a mixture of implicit and explicit newlines.  I have to think harder to predict what will happen.

Reply:
This now lives in `roff-rs`.